db/upgrade_migrations/20160723012349_upgrade_v062_to_v063.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+class UpgradeV062ToV063 < ActiveRecord::Migration[5.0]
+  def up
+    i = 1
+    Thredded::MessageboardGroup.order(created_at: :desc).each do |group|
```

Comment:
Perhaps we in a way that will work even if a user is running this migration on a previous version of Thredded (i.e. valid? returns true):

```ruby
Thredded::MessageboardGroup.where(
  name: Thredded::MessageboardGroup.group(:name).having('count(id) > 1').select(:name)
).group_by(&:name).each_value do |messageboard_groups|
   messageboard_groups.each_with_index do |messageboard_group, i|
     messageboard_group.update!(name: "#{messageboard_group.name}-#{i + 2}")
  end
end
```